docs/getting_started/running_a_workflow_locally.md

```diff
@@ -0,0 +1,94 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+# Running a workflow locally
+
+## Running a workflow in a local Python environment
```

Comment:
Depending on what we want to emphasize in this article, we should do one of two things:
* Include a note about the limitations of this approach in this section.
* Remove this section, make the entire article about running a workflow in a local cluster, and add a note about how you can run in a local Python environment as well, but that is not a recommended approach.